Formula/grass.rb

```diff
@@ -1,13 +1,13 @@
 class Grass < Formula
   desc "Geographic Resources Analysis Support System"
   homepage "https://grass.osgeo.org/"
-  revision 1
 
   stable do
-    url "https://grass.osgeo.org/grass64/source/grass-6.4.4.tar.gz"
-    sha256 "5ddba27b4e5495f602ee5249a07e287f342dd8e1422ea5d490c04311c731d274"
+    url "https://grass.osgeo.org/grass64/source/grass-6.4.5.tar.gz"
+    sha256 "f501da62807eb08efcb85820859fe5ade9bc392e20641b606273c956bb678f3e"
 
-    # Patches that files are not installed outside of the prefix.
+    # Patches to keep files from being installed outside of the prefix.
+    # Also, quick patch for compiling with clang (as yet, unreported issue)
     patch :DATA
```

Comment:
This has been fixed in upstream's `trunk` in http://trac.osgeo.org/grass/changeset/58434 (patch does not apply at all thanks to new files and all `*.cc` files having been renamed to `*.cpp`). A backport of this to the 6.4 branch has been requested in http://trac.osgeo.org/grass/ticket/2170 (already two years ago; the target milestone recently changed to 6.4.6, but the change hasn't made it into the upstream `releasebranch_6_4` branch yet).